sdk/python/feast/feature_server.py

```diff
@@ -86,8 +86,12 @@ def push(body=Depends(get_body)):
                 to = PushMode.OFFLINE
             elif request.to == "online":
                 to = PushMode.ONLINE
-            else:
+            elif request.to == "both":
```

Comment:
hm maybe slight preference for `online_and_offline` instead of `both` as that matches the existing enum and is more explicit?

Reply:
Fixed.